.gitlab/pipeline.yml

```diff
@@ -6,19 +6,8 @@
 
 include:
   - local: '.gitlab/includes/dockerhub_pipeline.yml'
```

Comment:
Sorry, I think this can also be removed from here since it's more or less a duplicate of the CircleCI configuration.